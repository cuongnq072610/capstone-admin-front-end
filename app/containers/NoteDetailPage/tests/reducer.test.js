import { fromJS } from 'immutable';
import noteDetailPageReducer from '../reducer';

describe('noteDetailPageReducer', () => {
  it('returns the initial state', () => {
    expect(noteDetailPageReducer(undefined, {})).toEqual(fromJS({}));
  });
});
