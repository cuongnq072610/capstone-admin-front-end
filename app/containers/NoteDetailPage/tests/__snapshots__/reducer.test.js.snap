// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`noteDetailPageReducer returns the initial state 1`] = `
Immutable.Map {
  "isLoading": false,
  "isLoadingUpdate": false,
  "isLoadingDelete": false,
  "note": Immutable.Map {},
  "error": Immutable.Map {},
  "message": Immutable.Map {},
}
`;
